lib/tiktoklikescroller.dart

```diff
@@ -109,13 +117,17 @@ class _TikTokStyleFullPageScrollerState
                   // Animate to next card
                   _state = DragState.animatingForward;
                 } else if ((_cardOffset >
-                            _containerSize.height /
-                                widget.swipePositionThreshold ||
-                        details.primaryVelocity! >
-                            widget.swipeVelocityThreshold) &&
-                    _cardIndex > 0) {
-                  // Animate to previous card
-                  _state = DragState.animatingBackward;
```

Comment:
Something's not quite right here. The deleted `_cardIndex > 0` condition means that the tests no longer pass and you can scroll to an index of -1. Do you want to check your logic?

Reply:
thanks! just learned how to run the tests and I'll defenitly check that out. I got an error when scrolled up on the first item, but it did not happen on my app so I wonder what went wrong...

Reply:
I found and fixed the issue. thanks for pointing that out! test are now passing